src/Utility/Inflector.php

```diff
@@ -144,7 +144,8 @@ class Inflector
         'goose' => 'geese',
         'foot' => 'feet',
         'foe' => 'foes',
-        'sieve' => 'sieves'
+        'sieve' => 'sieves',
+        'receive' => 'receives',
```

Comment:
Hmm.. "recieve" isn't a noun and "receives" is not a plural of "recieve".

Reply:
You're right and it totally slipped my mind when adding this. I just saw "InventoryReceife" in a freelance project and opened this.

Seems it should be a custom rule for this project. Sorry about that, I'll be more thoughtful next time. Should I create a revert pr?

Reply:
Yes please :)